contracts/lbp/LBPWrapper.sol

```diff
@@ -194,4 +257,20 @@ contract LBPWrapper {
     function feeAmountRequired() public view returns (uint256 feeAmount) {
         feeAmount = (amounts[0] * primeDaoFeePercentage) / HUNDRED_PERCENT;
     }
+
+    /**
+     * @dev checks if LBPWrapper holds the pool tokens or not
+     */
+    function holdsPoolTokens() public view returns (bool isHoldingPoolTokens) {
```

Comment:
Maybe call this `poolTokensInHolding`? holds sounds a bit off

Reply:
Also, maybe the frontend can just call this state to see if they can be retrieved?

Reply:
Yeah, we can do that too. Remove the function and directly, use it as modifier or inside the function.